sktm/patchwork.py

```diff
@@ -357,6 +357,11 @@ def get_header_value(self, patch_id, *keys):
 
         for key in keys:
             value = mbox_email.get_all(key, [''])
+            # Remove header folding
+            value = [
+                val.replace('\n', ' ').replace('\r', '').replace('\t', '')
```

Comment:
I think this might corrupt some unrelated cases. The proper way seems to be using something like this:

    val = re.sub("\r?\n[ \t]", " ", val)

Even though [RFC 2822 2.2.3. Long Header Fields](https://tools.ietf.org/html/rfc2822#section-2.2.3) says "Unfolding is accomplished by simply removing any CRLF that is immediately followed by WSP.", the practice seems to be to just replace WSP with space. WSP is either a space or a tab here.